### PR TITLE
About page and Setup page component

### PR DESCRIPTION
**Pull request by bryanling1**
image

image

### CODE REVIEW

File: client/src/views/AboutPage.tsx [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+  )
+}
+
+const MainWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`
+
+const WaterWrapper = styled.div`
```
Comment (linda9379) [inline client/src/views/AboutPage.tsx:143]: Is it possible to make that background a global or should we just have that variable in every file where we want this gradient to be

**Review by linda9379 — APPROVED** [review]
lgtm except one small question

**Comment by sandmich** [comment]
not sure how i feel about the pizza but it lgtm

**Review by sandmich — APPROVED** [review]

File: client/src/views/AboutPage.tsx (hunk 1 above)
Comment (bryanling1) [reply client/src/views/AboutPage.tsx:143]: You're right, we should add it as a global. Let's do this in future PRs

**Pull request merged**